docs/infrastructure.md

```diff
@@ -172,279 +170,213 @@ When you launch the graph backend for the first time,
+For example, if user `DBUSER` has been granted read/write access to database `my_db1` with the following command:
 
-        With GraphDB, there is no straightforward REST API call to _update_ a user's database access permissions without replacing the list of their existing database permissions (`"grantedAuthorities"`) entirely. 
-        
-        For example, if user `DBUSER` has been granted read/write access to database `my_db1` via a curl command with the following:
-        `{"grantedAuthorities": ["WRITE_REPO_my_db1","READ_REPO_my_db1"]}`
+```bash
+curl -X PUT --header 'Content-Type: application/json' -d '
+{"grantedAuthorities": ["WRITE_REPO_my_db","READ_REPO_my_db"]}' http://localhost:7200/rest/security/users/DBUSER -u "admin:NewAdminPassword"
+```
 
-        To grant `DBUSER` read/write access to a second database `my_db2` (while keeping the existing access to `my_db1`), 
-        when modifying http://localhost:7200/rest/security/users/DBUSER, 
-        you must specify all permissions - existing and new - in the same command 
-        since the existing permissions list will be overwritten:
-        `{"grantedAuthorities": ["WRITE_REPO_my_db1","READ_REPO_my_db1", "WRITE_REPO_my_db2","READ_REPO_my_db2"]}`
+!!! info
+    This command is also run as part of the [`graphdb_setup.sh`]() script for first-time GraphDB backend setup.
 
-        Similarly, to revoke `my_db1` access for `DBUSER`, so they only have access to `my_db2`, 
-        you would use the following permissions list:
-        `{"grantedAuthorities": ["WRITE_REPO_my_db2","READ_REPO_my_db2"]}`
+To grant `DBUSER` read/write access to a second database `my_db2` (while keeping the existing access to `my_db1`), 
+you would rerun the above `curl` command with _all_ permissions (existing and new) specified since the existing permissions list will be overwritten.
 
-        You can verify a user's settings at any time with the following:
-        ```bash
-        curl -u "admin:NewAdminPassword" http://localhost:7200/rest/security/users/DBUSER
-        ```
+So, your new provided permissions list would look as follows:
+```json
+{"grantedAuthorities": ["WRITE_REPO_my_db1","READ_REPO_my_db1", "WRITE_REPO_my_db2","READ_REPO_my_db2"]}
+```
 
-    !!! info "Managing user permissions using the GraphDB Workbench"
+Similarly, to revoke `my_db1` access so `DBUSER` only has access to `my_db2`, 
+you would use the following permissions list:
+```json
+{"grantedAuthorities": ["WRITE_REPO_my_db2","READ_REPO_my_db2"]}
+```
 
-        If you are managing multiple GraphDB databases, the web-based administration interface for a GraphDB instance, the Workbench, 
-        might be an easier way to manage user permissions than the REST API.
-        More information on using the GraphDB Workbench can be found [here](https://graphdb.ontotext.com/documentation/10.0/workbench-user-interface.html).
+!!! tip "Managing user permissions using the GraphDB Workbench"
```

Comment:
```suggestion
??? tip "Managing user permissions using the GraphDB Workbench"
```

Start collapsed, for the same reason - avoid getting too visually cluttered with boxes